src/mbgl/sprite/sprite_atlas.cpp

```diff
@@ -82,12 +88,12 @@ void SpriteAtlas::removeImage(const std::string& id) {
 
     Entry& entry = it->second;
 
-    if (entry.iconRect) {
-        bin.release(*entry.iconRect);
+    if (entry.iconBin) {
+//        shelfPack.release(*iconBin);
```

Comment:
Not currently, we're waiting on v2 / https://github.com/mapbox/shelf-pack-cpp/issues/5 which @bhousel is working on.